### DISCUSSION TITLE
Supabase Studio, same version different UI

### DISCUSSION THREAD

**Discussion by WilliamFalci** [supabase/supabase#4823] [Questions]
Hello guys,

I use supabase with docker from 2 different positions:

Home: Windows 11
Office: Linux Ubuntu

the supabase version is the latest released for both, but I have some differences into "Authentication" section

from office, I see only the "Policies" sub-section (look the screenshot) but from the one at home I see other 2 sub-sections

![image](https://user-images.githubusercontent.com/36926081/148362663-58ea1ad5-52d8-4418-bbd6-6420baebe4bd.png)

**Comment by MildTomato** [comment]
Could you confirm what browsers are being used?

**Reply by WilliamFalci** [reply]
> Could you confirm what browsers are being used?

Opera GX from home
Chrome from office
